STYLE.md

```diff
@@ -17,6 +17,7 @@
   includes both const and non-const references.
 * Function names using camel case starting with a lower case letter (e.g., "doFoo()").
 * Struct/Class member variables have a '\_' postfix (e.g., "int foo\_;").
+* Enum values using all CAPS (e.g., "EXTERNAL_ONLY").
```

Comment:
yeah, I'd do Pascal and create an issue to clean up all of the enum values which do not conform this rule (Those are mostly have HTTPX form)

@henna @htuch what would you say?